base/applications/mscutils/servman/lang/fr-FR.rc

```diff
@@ -111,17 +111,42 @@ CAPTION "Connexion"
 FONT 8, "MS Shell Dlg", 0, 0
 STYLE DS_SHELLFONT | WS_BORDER | WS_VISIBLE | WS_DLGFRAME | WS_SYSMENU | WS_GROUP | WS_TABSTOP
 BEGIN
+<<<<<<< HEAD
```

Comment:
Are you sure?

Reply:
Sure on what ? "Connexion" ?

Reply:
Serge was indirectly pointing out that you didn't correctly resolve the conflicts git showed to you.

Reply:
What a mess ...